corehq/apps/app_manager/xform.py

```diff
@@ -1895,7 +1895,8 @@ def get_action_path(action, create_subcase_node=True):
             path, subcase_node = get_action_path(action)
 
             open_case_block = CaseBlock(self, path)
-            subcase_node.insert(0, open_case_block.elem)
+            if subcase_node is not None:
+                subcase_node.insert(0, open_case_block.elem)
```

Comment:
Based on the sentry error and looking at the code I don't think `subcase_node` is `None`. `get_action_path` could only return `None` for `subcase_node` if:
* it was called with `create_subcase_node=False` which is no the case here
* `parent_node` was None which is also not the case since that would have generated an error before this one.

`subcase_node` for this error is a `WrappedNode` and it's `xml` attribute is `None` which is what's causing the error. I think it's `None` from this line:
```
parent_node = self.instance_node.find(
                    '/{x}'.join(action.repeat_context.split('/'))[1:]
                )

subcase_node = parent_node
```
Since `.find` can return a WrappedNode with `xml=None`: https://github.com/dimagi/commcare-hq/blob/b75e4470e940272568daedd52a1dc0c9d173abd0/corehq/apps/app_manager/xform.py#L191

I don't quite understand how that could be though since it would mean that the form is missing an instance node or a model node which also isn't the case for this form.

Reply:
Hrm, I see what you mean.

Whatever caused this, it's rare and didn't cause a user-facing problem. I'm going to close out this PR and will think about this more if it comes up again (which will now cause the build to fail due to https://github.com/dimagi/commcare-hq/pull/24473).